### ISSUE TITLE
Zelda BOTW randomly hangs since ver 1180

### ISSUE THREAD

**Issue by Labaman** [yuzu-emu/yuzu#9003]
Yuzu version 1184
Hanging happens randomly more during the autosave, but not just a couple of times just hung during the movement of the world as well as the time of the dialogue with the NPC.
**Note:** it is the game itself that hangs. The emulator menu is available and I can, for example, stop/restart the emulation or quit the application in the correct way. 
 LOGS:
[zelda_botw_logs.zip](https://github.com/yuzu-emu/yuzu/files/9687656/zelda_botw_logs.zip)

**Comment by liamwhite** [comment]
We have been seeing reports of this for a while. Does it happen with any yuzu version prior to 1180?

**Comment by Labaman** [comment]
I hadn't noticed. Sometimes gaming sessions in Zelda BOTW up to 4 hours - no hang-ups observed.

**Comment by Labaman** [comment]
@liamwhite, perhaps I can provide additional debugging information?

**Comment by liamwhite** [comment]
Is this specific to the Flatpak version? Do the AppImages from the [official releases](https://github.com/yuzu-emu/yuzu-mainline/releases) freeze as well?

**Comment by Labaman** [comment]
@liamwhite Yes, the same behavior is reproduced in the Appimage version (1186). Below are the logs taken after the game hangs, running in Appimage version. 
[zelda_yuzu_appimage.zip](https://github.com/yuzu-emu/yuzu/files/9692315/zelda_yuzu_appimage.zip)

**Comment by liamwhite** [comment]
Are you willing to compile the project for yourself and test if it works that way?

**Comment by Labaman** [comment]
Yes, let's.

**Comment by Labaman** [comment]
@liamwhite I'm sorry, maybe I misunderstood you - didn't you mean try to compile a specific test branch of the project? If so, tell me which one?
And while on 1188 the problem is still relevant.

**Comment by liamwhite** [comment]
I am hoping to see what the results are like when you compile the master branch - what you would get from `git clone --recursive https://github.com/yuzu-emu/yuzu.git` - and report back.

**Comment by Labaman** [comment]
I get the following error when building:
```
n destructor 'constexpr std::vector<_Tp, _Alloc>::~vector() [with _Tp = unsigned char; _Alloc = std::allocator<unsigned char>]',
    inlined from 'void InputCommon::InputFromNfc::OnChange()' at /home/wmw/github/yuzu/src/input_common/input_poller.cpp:738:9:
/usr/include/c++/12.2.0/bits/stl_vector.h:730:22: error: 'status.Common::Input::CallbackStatus::camera_status.Common::Input::CameraStatus::data.std::vector<unsigned char>::<anonymous>.std::_Vector_base<unsigned char, std::allocator<unsigned char> >::_M_impl.std::_Vector_base<unsigned char, std::allocator<unsigned char> >::_Vector_impl::<anonymous>.std::_Vector_base<unsigned char, std::allocator<unsigned char> >::_Vector_impl_data::_M_start' may be used uninitialized [-Werror=maybe-uninitialized]
  730 |         std::_Destroy(this->_M_impl._M_start, this->_M_impl._M_finish,
      |         ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  731 |                       _M_get_Tp_allocator());
      |                       ~~~~~~~~~~~~~~~~~~~~~~
/home/wmw/github/yuzu/src/input_common/input_poller.cpp: In member function 'void InputCommon::InputFromNfc::OnChange()':
/home/wmw/github/yuzu/src/input_common/input_poller.cpp:735:45: note: 'status' declared here
  735 |         const Common::Input::CallbackStatus status{
      |                                             ^~~~~~
In destructor 'constexpr std::_Vector_base<_Tp, _Alloc>::~_Vector_base() [with _Tp = unsigned char; _Alloc = std::allocator<unsigned char>]',
    inlined from 'constexpr std::vector<_Tp, _Alloc>::~vector() [with _Tp = unsigned char; _Alloc = std::allocator<unsigned char>]' at /usr/include/c++/12.2.0/bits/stl_vector.h:733:7,
    inlined from 'void InputCommon::InputFromNfc::OnChange()' at /home/wmw/github/yuzu/src/input_common/input_poller.cpp:738:9:
/usr/include/c++/12.2.0/bits/stl_vector.h:367:31: error: 'status.Common::Input::CallbackStatus::camera_status.Common::Input::CameraStatus::data.std::vector<unsigned char, std::allocator<unsigned char> >::<unnamed>.std::_Vector_base<unsigned char, std::allocator<unsigned char> >::_M_impl.std::_Vector_base<unsigned char, std::allocator<unsigned char> >::_Vector_impl::<anonymous>.std::_Vector_base<unsigned char, std::allocator<unsigned char> >::_Vector_impl_data::_M_end_of_storage' may be used uninitialized [-Werror=maybe-uninitialized]
  367 |                       _M_impl._M_end_of_storage - _M_impl._M_start);
      |                       ~~~~~~~~^~~~~~~~~~~~~~~~~
/home/wmw/github/yuzu/src/input_common/input_poller.cpp: In member function 'void InputCommon::InputFromNfc::OnChange()':
/home/wmw/github/yuzu/src/input_common/input_poller.cpp:735:45: note: 'status' declared here
  735 |         const Common::Input::CallbackStatus status{
      |                                             ^~~~~~
cc1plus: all warnings being treated as errors
[23/243] Building CXX object src/shader_recompiler/CMakeFiles/shader_recompiler.dir/backend/glsl/emit_glsl.cpp.o
ninja: build stopped: subcommand failed.
```

**Comment by liamwhite** [comment]
Unfortunately I am unable to reproduce this compilation error with gcc 12.2.0 / libstdc++ 12.2.0 on Kali. I am not sure how to reproduce your specific setup, there may be a distro patch that is causing me to not see the error.

**Comment by Labaman** [comment]
@liamwhite My system is a standard Arch installation.
The compilation using clang did finish without errors. 
Tested for 3 hours - tried to use loads/saves as often as possible - no hangs.

**Comment by Labaman** [comment]
1193 The game still periodically hangs. Please do something - it's impossible to play with this error.

**Comment by liamwhite** [comment]
Using flatpak 1193, I started a with a fresh save file and completed the Great Plateau quest up to and including the Magnesis shrine. I did not experience any hangs. Mesa RADV, 6700XT, Ryzen 9 5950X. I'm not sure how I can reproduce the issue.

**Comment by shelvacu** [comment]
I'm seeing this on both a steam deck (flatpak) and windows 11/rx 6700 xt. Using both DLCs and update 1.6.0.

It's frustratingly random, I can't pin it down to one thing. Someone else mentioned autosaves, but that doesn't seem right as I've had the freeze happen on the Game Over screen.

It maybe seemed to happen more often in gerudo valley? If you want I can send you my save

**Comment by Labaman** [comment]
The problem is not a specific location: at the moment I observe the problem in the village of Lurelin and on the island of "Evening", previously in Hateno, Zoro's possessions, Kakariko. In other words, in the "Big" world outside the old location. 
If necessary, I can also send my saves. 
In the master branch, I was never able to reproduce the problem by continuing to play using the same save. 
Archlinux 
Nvidia 515.76-1.
Zelda BOTW update 1.6.0 + DLC 
By the way, a similar problem was previously in Ryujinx: the game randomly hung up, but there were no hangs on the Great Plateau. 
https://github.com/Ryujinx/Ryujinx/issues/2314
There, as far as I remember, was a problem with GPU memory errors. Perhaps this problem was reproduced in Yuzu?

**Comment by liamwhite** [comment]
I loaded the save from my console and played for a few hours:
- Battled some enemy camps
- Completed some NPC quests (Kass)
- Unlocked a shrine
- Ran around the world
- Fast traveled around the world
- Used the BLSS glitch to try to force world loads, which don't occur to anywhere near the extent they do on console

None of these resulted in a hang for me, with 1.6.0 + DLC on Flatpak mainline 1193.

**Comment by Labaman** [comment]
Did the following:
Selected delete all in the context menu of the title. 
After that, re-installed all the updates and DLC.
For several days I played for several hours, visiting the same locations, and doing the same actions, in which previously the problem was reproduced quite often - it did not hang.
But today, after installing 1199, and another attempt to "conquer" the Evening island, I left the character just standing on the mountain in the center of the island for about 5-10 minutes.  The game sound was playing, so I thought everything was finally OK, but when I tried to control the character, I realized that the game had hung up (while the environmental sounds and music in the game continued to play). I tried pausing and resuming the emulation, but it didn't work. I stopped the emulation through the appropriate menu item.

Logs:
[log_BOTW.zip](https://github.com/yuzu-emu/yuzu/files/9792482/log_BOTW.zip)
Saves:
[01007EF00011E000.zip](https://github.com/yuzu-emu/yuzu/files/9792490/01007EF00011E000.zip)

**Comment by MikeOwaR** [comment]
I am also getting this issue. Logs attached

[botw_yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/9844151/botw_yuzu_log.zip)

**Comment by MikeOwaR** [comment]
Same issue here. Game randomly hangs

[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/9844156/yuzu_log.zip)

**Comment by zhgjerry** [comment]
Repeatable with steam deck flatpak version 1207. It's randomly drop to 0fps, emulator is still running, even stay in the start screen might trigger it (not actually load game save yet).

**Comment by zhgjerry** [comment]
might be same issue as #9174.
Confirmed same issue happen on P5R, Bayo3 on steam deck, easy to repeat for idle on first screen in desktop mode.
Looks like disable "mutlticore CPU emulation" fix the issue. (performance will drop).

**Comment by OranShuster** [comment]
Also having freeze issues with BoTW
From the logs - Userspace panic

Windows 11
RTX 3070
I7 12th gen
Vulkan + Fast GPU + Multicore CPU emulation

[yuzu_log_userspace_panic.txt](https://github.com/yuzu-emu/yuzu/files/10117891/yuzu_log_userspace_panic.txt)

**Comment by Jaynator495** [comment]
> might be same issue as #9174. Confirmed same issue happen on P5R, Bayo3 on steam deck, easy to repeat for idle on first screen in desktop mode. Looks like disable "mutlticore CPU emulation" fix the issue. (performance will drop).

I can also confirm the same behavior on steam deck, with the addition of Mario Kart 8 Deluxe to the games list that it occurs on. 

Attached is logs for both BOTW and MK8D crashes
[botw VK_ERROR_SURFACE_LOST_KHR.txt](https://github.com/yuzu-emu/yuzu/files/10200998/botw.VK_ERROR_SURFACE_LOST_KHR.txt)
[botw Userspace PANIC.txt](https://github.com/yuzu-emu/yuzu/files/10200997/botw.Userspace.PANIC.txt)
[mk8d Userspace PANIC.txt](https://github.com/yuzu-emu/yuzu/files/10200999/mk8d.Userspace.PANIC.txt)

**Comment by liamwhite** [comment]
Should be fixed by #9474
